modules/apps/change-tracking/change-tracking-rest-impl/rest-openapi.yaml

```diff
@@ -411,6 +411,52 @@ paths:
                                 type: array
                     description: ""
             tags: ["Entry"]
+    "/collections/{collectionId}/entries/{entryId}/affected-entries":
```

Comment:
Do you really need to scope it under collections? does not an entryId identify univocally an entry inside a collection?

Reply:
Well entryId could be enough in normal cases, but I didn't want to modify de original queries we used in our former rest module. 
Later we may simplify it when Change Tracking API reaches it's final form. Right now, It will be enough for us.